content/blog/entries/automate-github-for-more-than-CI CD/contents.lr

```diff
@@ -0,0 +1,144 @@
+title: Automate GitHub for more than CI/CD
+---
+categories:
+community
+cc-search
+open-source
+---
+author: zackkrida
+---
+pub_date: 2020-08-26
+---
+body:
+
+> *Get started using GitHub bots and actions for community management and repository health.*
+
+In late 2018, in the midst of being acquired by Microsoft, GitHub [launched Github Actions](https://github.blog/2018-10-16-future-of-software/) into public beta, allowing users to run code on the popular development platform for the first time. With a straightforward `YAML` configuration syntax and the power of Microsoft's Azure cloud, GitHub actions quickly rose to compete with existing Continuous Integration (CI) and Continuous Deployment (CD) platforms like **Circle CI** and **Travis CI**. GitHub actions made it easier than ever for developers to test and deploy software in the cloud, but from the beginning GitHub had bigger plans for the service.
```

Comment:
GitHub actions --> GitHub Actions (GitHub capitalizes the A), this applies everywhere.